docusaurus/docs/adding-images-fonts-and-files.md

```diff
@@ -9,6 +9,12 @@ You can **`import` a file right in a JavaScript module**. This tells Webpack to
 
 To reduce the number of requests to the server, importing images that are less than 10,000 bytes returns a [data URI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) instead of a path. This applies to the following file extensions: bmp, gif, jpg, jpeg, and png. SVG files are excluded due to [#1153](https://github.com/facebook/create-react-app/issues/1153).
 
+To override the default set the environment variable `IMAGE_INLINE_SIZE_LIMIT`. For example:
```

Comment:
Missing comma. _To override the default,_

Nit: Specify what format (bits, bytes, MB?) the value is set to. Although, it's mentioned in the paragraph above, it's easier to get here.

Reply:
PR is stale, which I assume means there's no interest in the change.

Reply:
What needs to be done to help move this along?

Reply:
Find someone on the core team willing to take a look.

Reply:
I'd like to get rid of the merge conflicts and build failures before doing so. Is it okay if I send a PR to your repo/branch? (https://github.com/peterbe/create-react-app/tree/3437-environment-variable-to-disable-inlining-images)

Reply:
Yes please go for it!

Reply:
Apparently the CI fails for `master` right now  :-) Hopefully that's not a show-stopper

Reply:
@Timer would you recommend rebasing the PR to a release branch, which passes CI tests? I'm not familiar with the official policy here. I can see is most PRs are failing CI because they stem from a failing commit from master history, which is a bit confusing.